components/Player/Player.tsx

```diff
@@ -58,7 +58,7 @@ const Player: React.FC<IPlayerProps> = ({
   const { url, previewUrl, transcripts, duration } = currentTrack;
   const currentTrackUrl = previewUrl || url;
   const transcript = transcripts?.find(
-    (t) => !!['vtt', 'srt', 'json'].find((n) => t.type.includes(n))
+    (t) => !!['vtt', 'srt', 'x-subrip', 'json'].find((n) => t.type.includes(n))
```

Comment:
looks good - the docs have these as options:
https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#attributes
```ext/plain, text/html, text/vtt, application/json, application/x-subrip```
So looking for subrip and srt is good!